scripts/retrigger_file_validation/usage.md

```diff
@@ -0,0 +1,15 @@
+# How to revalidate failed file validation in Ingest
```

Comment:
Please rename the file to readme.md so it is shown by github in the [folder view](https://github.com/ebi-ait/hca-ebi-dev-team/tree/ops-670_retrigger_failed_file_validation/scripts/retrigger_file_validation).

Reply:
Done. Pushed